[docs] [reference] compression types for yb-tserver (#12062)
yugabyte/yugabyte-db: docs/content/preview/reference/configuration/yb-tserver.md

### Network compression

Use the following flag to select the compression type.

##### --compression_type

The valid compression types are `Snappy`, `Zlib`, `LZ4`, and `NoCompression`.

Default: `Snappy`

{{< note title="Notes" >}}

> - If an invalid option is selected, the cluster will not come up.
> - If this flag is changed, it will take effect after the cluster nodes are restarted.

Reviewer: If you select... If you change...
```suggestion
- If you select an invalid option, the cluster will not come up.
- If you change this flag, the change takes effect after you restart the cluster nodes.
```
